### ISSUE TITLE
Build a isBitcoinInvoiceExpired computed field for Payments

### ISSUE THREAD

**Issue by otech47** [setlife-network/trinary#623]
We will build this into the set of subfields of the PaymentType

```
Payment {
    id: Int
    amount: Int
    ...
    isBitcoinInvoiceExpired: Boolean
}
```

Check:

- If `external_uuid_type = 'bitcoin'`
- If `external_uuid` has a value
- Take `external_uuid` and ask BTCPayserver if the invoice is expired
- Return true/false

**Comment by otech47** [comment]
> Should I build a new Query getCheckoutUrl that just returns the checkout Url if the invoice has not expired

@rishkwal that could be useful and I imagine it could be implemented alongside or similarly to this issue ^

**Comment by otech47** [comment]
Something like

```
PaymentType {
    checkoutUrl: String
}
```

Fails if:
- `external_uuid_type != 'bitcoin'`
- `external_uuid` is blank

Automatically creates a new invoice if expired?

If you find a clever way to consolidate these 2 fields that might be nice, but not strictly necessary it would be fine if they were separate but give it some thought

Feel free to create a new issue for this instead

**Comment by rishkwal** [comment]
resolved by #627
